src/irmin/import.ml

```diff
@@ -146,3 +146,106 @@ let shuffle state arr =
   let len = Array.length arr in
   aux (len - 1);
   ()
+
+module Small_map = struct
```

Comment:
Might be worth adding a small comment to document the intended usage of this module (eg only intended for "small" key sets and give a sense of what "small" is).